src/add-to-app/android/project-setup.md

```diff
@@ -59,6 +58,7 @@ android {
 <div class="tab-pane active" id="with-android-studio" role="tabpanel" aria-labelledby="with-android-studio-tab" markdown="1">
 
 ### Integrate with Android Studio
+
```

Comment:
These `{:.no_toc}` entries can't have the space before them, as otherwise the class won't apply to the header. Mind reverting these changes?

Reply:
My Markdown linter is doing this on save and I can't turn it off, but I did fix it from making this change. I'll revise.

Reply:
With the 11ty work, these will be on the same line as the header, so hopefully the linter can handle that.

I turned off fixing lints of save for mine, as it's too disruptive and not necessarily all correct.